platform/darwin/src/MGLTileSet.h

```diff
@@ -14,6 +14,16 @@ typedef NS_ENUM(NSUInteger, MGLTileSetScheme) {
  configuration for those strings. It can be passed to an `MGLVectorSource` or
  `MGLRasterSource` instead of an `NSURL` representing a TileJSON URL to create a 
  source.
+ 
+ Tile URL template strings are a loose open standard for referring to custom 
+ locations from which to download tiles, whether vector or raster. An example 
+ template string is `http://myserver.com/tiles/{z}/{x}/{y}.pbf`.
+ 
+ In the current implementation, only the first tile URL template string specified
```

Comment:
If this fact is subject to change, as an implementation detail, we should leave it out of documentation. If it won't change, we should allow only the first template to be set and ignore any other templates that happen to be set in the style. That would simplify the API a little.